src/vs/workbench/services/languageRuntime/common/languageRuntimePlotClient.ts

```diff
@@ -209,6 +118,9 @@ export class PlotClientInstance extends Disposable implements IPositronPlotClien
 	onDidRenderUpdate: Event<IRenderedPlot>;
 	private readonly _renderUpdateEmitter = new Emitter<IRenderedPlot>();
 
+	/**
+	 * Event that fires when the plot wants to display itself.
```

Comment:
I've reworded it. The plot doesn't want to display itself, it's the runtime that sent an event to show the plot. I think we only care that the plots view shows or switches to showing this plot. We could also make the editor tab associated with this plot the active editor but I think it would be good to see if users want that.